apis/v1alpha1/cluster_types.go

```diff
@@ -255,7 +258,10 @@ type IngressConfig struct {
 	// +optional
 	UI *Ingress `json:"ui,omitempty"`
 
-	// TODO: Add other ingress config for SQL and GRPC here when implemented
+	// (Optional) SQL port to be exposed via ingress
```

Comment:
nit: this isn't just a port. Maybe something like `(Optional) Ingress options for SQL connections` ?

Reply:
Done